metron-docker/kafkazk/bin/wait-for-it.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+#   Use this script to test if a given TCP host/port are available
```

Comment:
It feels like we should have some type of packaging step in order to accomplish the following:
- Copy resources to final structure for the deploy
- Perform parameterization
- Eliminate file duplication